src/jwt-utils.ts

```diff
@@ -1,7 +1,12 @@
 import jwt, { Secret } from 'jsonwebtoken';
 
-export function generateToken(payload: string | object | Buffer) {
-  return jwt.sign({ payload }, process.env.JWT_KEY as Secret, { expiresIn: 60 * 15 });
+export function generateToken(payload: string | object | Buffer, extendDuration: boolean) {
+  //1 week in seconds
+  const week = 60 * 60 * 24 * 7;
```

Comment:
Duas sugestões que vão ao encontro do que o Clean Code fala sobre comentários:

```suggestion
  const oneWeekInSeconds = 60 * 60 * 24 * 7;
```

```suggestion
  const extendedExpiration = 60 * 60 * 24 * 7;
```

A segunda tem uma abordagem de nomear o valor pelo que ele representa, e não pela quantidade. Eu gosto mais da segunda.

Reply:
Tem uma forma de conciliar tbm, que seria `extendedExpirarionInSeconds`

Reply:
Feito